components/custom-network/theme-colors.tsx

```diff
@@ -51,16 +45,33 @@ const ThemeStringInput = ({label, text, error, onChange}) => {
           d-flex flex-row align-items-center border-radius-8  cursor-pointer
           `}
           >
-          <input
-            type="text"
-            className={`custom-text-input ${(error && "is-invalid") || ""} form-control`} 
-            name={label}
-            id={label}
-            value={text}
-            ref={themeStringInput}
-            onChange={handleChange}
-            onClick={copyContent}
-          />
+          {/* The update key is used to force a rerender on the input component when a color input changed. */}
+          <div key={updateKey}>
+            <input
+              type="text"
+              className={`caption custom-text-input ${(error && "is-invalid") || ""} form-control`} 
+              name={label}
+              id={label}
+              defaultValue={initialText}
+              ref={themeStringInput}
+              onChange={onChange}
+              
+              /*
+              The changes are applied on blur and paste 
+              to avoid funky behavior when manually editing
+              the text
+              */
+              onPaste={onPaste}
+              onBlur={onBlur}
+
+              /*
+              As the primary use case is to just paste values in. 
+              It is more convenient to select all for a quick paste.
+              Instead of having to select everything manually.
+              */
+              onFocus={(evt)=>evt.target.select()} 
```

Comment:
When the input is selected it no longer copies the content directly, but it still selects the whole text, that the user can instantly paste their copied Theme.